### ISSUE TITLE
"test story" at test case group or test plan level 

### ISSUE THREAD

**Issue by carolinerosin** [Jungyubw/newTCAMT#107]
I think it would be nice to have the "Test Story" tab (maybe the name should change, but that is a detail) at TestCaseGroup and TestPlan level.
(I have some examples in LRI I can show if needed)

**Comment by Jungyubw** [comment]
Of course we can do, but let's ask to Rob first. 
@robnist, what do you think of this?

**Comment by robnist** [comment]
Yes. We technically have this is old TCAMT where we place this information into the Description I think and then this gets carried over into the TEST as the "Test Story" at the Plan and Group level.

So yes, and yes the tab name can be changed to Description or something else at the Plan and Group level.

**Comment by Jungyubw** [comment]
Okay. I will make new Ticket for this requirements.
